ui/src/types/variables.ts

```diff
@@ -0,0 +1,7 @@
+export {VariableProperties} from 'src/client'
+import {Variable as GenVariable} from 'src/client'
+import {Label} from 'src/types'
+
+export interface Variable extends GenVariable {
+  labels?: Label[]
```

Comment:
labels need to be required.

Reply:
Why? They aren't required in the swagger file:

https://github.com/influxdata/influxdb/blob/master/http/swagger.yml#L8817

Reply:
The reason for setting labels here (and basically having them do the same thing) is to ensure that the type is declared to the same type. That is, since the default for `Variable` labels are the `Labels` generated in the oats file, they are not the same as the ones that are assigned to the `Labels` in the `src/types` file

Reply:
But now that we've actually stopped customizing the Label type and are simply exporting it, it seems like this is pretty unnecessary. I'm gonna revert it to the original `export {Variable} from src/client` if the linter returns no errors